Prox/Prox/PlaceDetails/PlaceDetailViewController.swift

```diff
@@ -53,5 +96,53 @@ class PlaceDetailViewController: UIViewController {
         super.didReceiveMemoryWarning()
         // Dispose of any resources that can be recreated.
     }
+
+
+    func pageControlDidPage(sender: AnyObject) {
+        let pageSize = imageCarousel.bounds.size
+        let xOffset = pageSize.width * CGFloat(pageControl.currentPage)
+        imageCarousel.setContentOffset(CGPoint(x: xOffset, y: 0), animated: true)
+    }
     
 }
+
+extension PlaceDetailViewController: UICollectionViewDataSource {
+    func numberOfSections(in collectionView: UICollectionView) -> Int {
+        return 1
+    }
+
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return place.photoURLs?.count ?? 1
+    }
+
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: CellReuseIdentifier, for: indexPath) as! ImageCarouselCollectionViewCell
+        if let photoURLs = place.photoURLs,
+            let photoURL = URL(string: photoURLs[indexPath.item]) {
+            cell.imageView.setImageWith(photoURL)
```

Comment:
it's in `UIImageView_AFNetworking` - an extension provided inside `AFNetworking`